### ISSUE TITLE
Tile render order for hexagonal maps does not work.

### ISSUE THREAD

**Issue by Asmageddon** [bjorn/tiled#1811]
It appears to work as Right Down regardless of the setting. Can be seen on the hexagonal-mini.tmx example

**Comment by bjorn** [comment]
Yeah, [as documented here](http://doc.mapeditor.org/en/latest/reference/tmx-map-format/#map) this property is currently only supported for orthogonal maps. It could of course be implemented for hexagonal maps as well. Do you actually have hexagonal tiles that should be rendered in a different order?

**Comment by Asmageddon** [comment]
Hexagonal tilesets don't tile properly with even tile sizes, which means that Right-Down creates these nasty little issues, such as:

![Example of issue](https://i.imgur.com/KM8iHal.png)

...although now that I think about it, Left-Down would have the same issue. Maybe Up directions would work right?
